.gitignore

```diff
@@ -7,6 +7,7 @@ tags
 *.fail.go
 vendor
 .DS_Store
+core.
```

Comment:
What's this?

Reply:
In fact, I don't know which step this is from, but it does appear in the directory, so I added it to the ignore.